files/synchronize.py

```diff
@@ -407,10 +407,10 @@ def main():
     if ssh_args:
       ssh_opts = '%s %s' % (ssh_opts, ssh_args)
 
-    if source.startswith('rsync://') and dest.startswith('rsync://'):
+    if source.startswith('"rsync://') and dest.startswith('"rsync://'):
```

Comment:
I find these two a bit curious. Could you explain what you found that the " character is required here?